controllers/phone-numbers/removePhoneNumberFromTwilio.js

```diff
@@ -6,7 +6,13 @@ const { deletePhoneNumberFuncTwilio } = require('../../services/twilio.service')
 const removePhoneNumberFromTwilio = async (req, res, next) => {
 	try {
 		const { phoneNumber } = req.params;
-
+		const user = req.user;
+		if (user.role !== 'CRM_ADMIN' && user.role !== 'CRM_COMPANY_ADMIN' && user.role !== 'CRM_MANAGER') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth